contrib/win32/install/server.wxs

```diff
@@ -0,0 +1,70 @@
+    <Fragment>
+        <!-- KeyPath is necessary for multi-file components to identify the key file - preferrably versioned. -->
+        <ComponentGroup Id="Server" Directory="INSTALLFOLDER">
+            <ComponentGroupRef Id="Shared" />
+            <Component>
+                <File Name="sftp-server.exe" KeyPath="yes" />
+                <File Name="sftp-server.pdb" />
+            </Component>
+            <Component>
+                <File Name="ssh-shellhost.exe" KeyPath="yes" />
+                <File Name="ssh-shellhost.pdb" />
+            </Component>
+            <Component>
+                <File Id="sshd.exe" Name="sshd.exe" KeyPath="yes" />
+                <File Name="sshd.pdb" />
+                <RegistryKey Root="HKLM" Key="SOFTWARE\OpenSSH" ForceCreateOnInstall="yes">
+                    <PermissionEx Sddl="O:BAG:SYD:P(A;OICI;KR;;;AU)(A;OICI;KA;;;SY)(A;OICI;KA;;;BA)" />
+                    <!-- ssh-agent-associated key should only be created if the Server feature is installed. -->
+                    <RegistryKey Key="agent" ForceCreateOnInstall="yes">
+                        <PermissionEx Sddl="O:BAG:SYD:P(A;OICI;KA;;;SY)(A;OICI;KA;;;BA)" />
+                    </RegistryKey>
+                </RegistryKey>
+                <ServiceInstall
+                    Name="sshd"
+                    DisplayName="OpenSSH SSH Server"
+                    Description="OpenSSH is a connectivity tool for remote login that uses the SSH protocol. It encrypts all traffic between client and server to eliminate eavesdropping, connection hijacking, and other attacks."
+                    Start="auto"
+                    Type="ownProcess"
+                    Interactive="no"
+                    ErrorControl="critical"
+                    Vital="yes">
+                    <util:ServiceConfig
+                        ResetPeriodInDays="1"
+                        FirstFailureActionType="restart"
+                        SecondFailureActionType="restart"
+                        ThirdFailureActionType="restart"
+                        />
+                </ServiceInstall>
+                <ServiceControl
+                    Id="ControlSshd"
+                    Name="sshd"
+                    Start="install"
+                    Stop="both"
+                    Remove="uninstall" />
+                <firewall:FirewallException
+                    Id="sshd_allow"
+                    Name="OpenSSH SSH Server Preview (sshd)"
+                    Description="Inbound rule for OpenSSH SSH Server (sshd)"
+                    Program="[#sshd.exe]"
+                    Protocol="tcp"
+                    Port="22"
+                    Scope="any"
+                    />
+            </Component>
+            <Component>
+                <File Name="sshd_config_default">
+                    <PermissionEx Sddl="O:BAG:SYD:PAI(A;;FA;;;SY)(A;;FA;;;BA)" />
```

Comment:
We can have read permissions to the authorized users.

Reply:
This was copied from the scripts at the time of writing. Have they changed?